addons/event_crm/views/crm_lead_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="crm_lead_view_form_inherited" model="ir.ui.view">
+        <field name="name">crm.lead.form.inherited</field>
```

Comment:
guidelines (see the other comment on inherited views guidelines)

Reply:
done